shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -221,6 +241,7 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
             }
         }
         if (object.hasAction(Action.INCREASE) || object.hasAction(Action.DECREASE)) {
+            // TODO(jonahwilliams): support AccessibilityAction.ACTION_SET_PROGRESS once SDK is updated.
```

Comment:
Can you just copy the Action's ID like we did for ACTION_SHOW_ON_SCREEN to make this work now? Also, how is that action exposed to the user?

Reply:
It is exposed as a special action in the local context menu - but since it is a newer action it doesn't have a hardcoded resource id, just a static field